packages/inventory/src/containers/CreateServicePoint/index.tsx

```diff
@@ -26,9 +23,7 @@ const ServicePointsAdd = (props: ServicePointAddTypes) => {
     openSRPBaseURL: baseURL,
     instance: FormInstances.EUSM,
     hidden: commonHiddenFields,
-    successURLGenerator: (payload?: LocationUnit) =>
-      `${INVENTORY_SERVICE_POINT_PROFILE_VIEW}/${payload?.id}`, // todo if payload is missing
-    cancelURLGenerator: () => INVENTORY_SERVICE_POINT_LIST_VIEW,
```

Comment:
I think this is the correct change

Reply:
Auto-merging causes this issue, 
Will refactor this in next push